Sources/Clients/SecureStorageClient/SecureStorageClient+Live.swift

```diff
@@ -135,39 +135,26 @@ extension SecureStorageClient: DependencyKey {
 			try await deleteProfileHeader(id)
 		}
 
-		@Sendable func loadDeviceIdentifier() async throws -> UUID {
```

Comment:
remove generate and save, part of `load`, now load only... loads...

Reply:
Yeah, so where do we generate a new one, if needed?

Reply:
inside `checkIfDeviceOwnsProfileSnapshot`